src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -615,16 +615,16 @@ final class HTTPClient: Client {
 		})
 	}
 	
-	func getDCCOnboardedCountries(
+	func validationOnboardedCountries(
```

Comment:
All other "caching" requests are part of `CachingHTTPClient`.
For consistency reasons, can we also move this there?

Reply:
I implemented it in the "normal" client, because:
1. The CachingHTTPClient does not handle the hole fake request stuff so far. So I had to copy a lot from the normal client over to the CachingHTTPClient
2. The error handling of the tech specs is more familiar with the error handling of the normal client. If I move this to the CachingHTTPClient, the error handling would be much more different then the other calls.
3. We have a ticket to refactor the client and the caching client to make these clients more beautiful. We could clean up this here in that ticket, too.